generation/core/generator.js

```diff
@@ -7,13 +7,14 @@ const fs = require("fs");
 
 const { methodNameToSnakeCase } = require("../helpers");
 let globalFunctionUsage = {};
-module.exports = function({
+module.exports = function ({
```

Comment:
Could you please make it a bit more readable by creating a named function and exporting it, instead of exporting an anonymous one. Sorry for nitpicking, I just can't get used to this javascript jungle magic. :(

Reply:
sure, what would be a good name for a function that returns a code generator?

Reply:
getGenerator ?